unit_test/live_model_test.cc

```diff
@@ -116,6 +117,29 @@ BOOST_AUTO_TEST_CASE(live_model_ranking_request) {
   BOOST_CHECK_EQUAL(status.get_error_msg(), "");
 }
 
+BOOST_AUTO_TEST_CASE(live_model_ranking_request_pdf_passthrough) {
```

Comment:
>live_model_ranking_request_pdf_passthrough [](start = 21, length = 42)

Can we add a sanity check that at least verifies the probability responses match the expected PASSTHROUGH_PDF_MODEL probabilities?